src/BenchmarksApps/TechEmpower/Minimal/Program.cs

```diff
@@ -27,9 +29,12 @@
 
 app.MapGet("/db", async (Db db) => await db.LoadSingleQueryRow());
 
+var createFortunesTemplate = RazorSlice.ResolveSliceFactory("/Templates/Fortunes.cshtml");
 app.MapGet("/fortunes", async (HttpContext context, Db db) => {
     var fortunes = await db.LoadFortunesRows();
-    await FortunesTemplate.Render(fortunes, context.Response);
+    var template = (RazorSliceHttpResult<List<Fortune>>)createFortunesTemplate();
```

Comment:
can this be invoked once? per app?

Reply:
No the template instance has to be created per request. I did an experiment to pool the instances and it didn't result in any improvement.